AnnService/src/Core/VectorIndex.cpp

```diff
@@ -294,6 +294,42 @@ VectorIndex::DeleteIndex(ByteArray p_meta) {
 }
 
 
+ErrorCode
+VectorIndex::MergeIndex(const char* p_indexFilePath)
+{
+    std::string folderPath(p_indexFilePath);
+    if (!folderPath.empty() && *(folderPath.rbegin()) != FolderSep) folderPath += FolderSep;
```

Comment:
else ？

Reply:
not need for "else" since if the folderPath is empty,  iniReader.LoadIniFile will return ErrorCode::FailedOpenFile error.